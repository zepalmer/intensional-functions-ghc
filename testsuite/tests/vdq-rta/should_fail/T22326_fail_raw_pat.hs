{-# LANGUAGE ExplicitNamespaces #-}
{-# LANGUAGE RequiredTypeArguments #-}

module T22326_fail_raw_pat where

f :: forall (a :: k) -> ()
f x = ()