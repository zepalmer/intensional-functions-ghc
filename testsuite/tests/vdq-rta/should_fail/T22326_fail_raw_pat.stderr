
T22326_fail_raw_pat.hs:7:3: error: [GHC-88754]
    • Ill-formed type pattern: x
      Expected a type pattern introduced with the ‘type’ keyword.
    • In an equation for ‘f’: f x = ()
